go.mod

```diff
@@ -16,6 +16,7 @@ require (
 	github.com/gorilla/mux v1.7.3
 	github.com/heroku/docker-registry-client v0.0.0-20190909225348-afc9e1acc3d5
 	github.com/mattermost/mattermost-cloud v0.7.0
+	github.com/mattermost/mattermost-server v5.11.1+incompatible
```

Comment:
@jwilander Why is the import of this version needed?

Reply:
I didn't pick that version specifically. I just ran `go mod vendor` to fix the lint issues so I could get this merged.